validator/packaging/validator.toml.example

```diff
@@ -0,0 +1,17 @@
+#
+# Sawtooth Lake -- Validator Configuration
+#
+
+# This file should exist in the defined config directory and allows
+# validators to be configured without the need for command line options.
+# It is important to note that public_uri must be set by either the config or
+# the command line.
+
+# The following is a possible example.
+
+network_endpoint = "tcp://127.0.0.1:8800"
```

Comment:
Was thinking about this file in this context: https://github.com/hyperledger/sawtooth-core/blob/8f1ea3a22f0fcf1ccbe994c05c3b4469edaab026/validator/packaging/path.toml.example